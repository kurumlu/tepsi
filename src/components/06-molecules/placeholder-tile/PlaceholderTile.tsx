import * as React from 'react';
import { FunctionComponent } from 'react';

export const PlaceholderTile: FunctionComponent = () => (
  <div className="as-m-product-tile">
    <svg
      style={{ width: '100%', height: 'auto' }}
      preserveAspectRatio="none"
      role="img"
      viewBox="0 0 316 620">
      <path fill="#fff" d="M-1-1h1002v1002H-1z" />
      <g fill="#f3f3f3">
        <rect width="316" height="474" rx="5" ry="5" />
        <rect width="186" height="20" y="480" rx="0" ry="0" />
        <circle cx="6" cy="536" r="6" />
        <circle cx="22" cy="536" r="6" />
        <circle cx="38" cy="536" r="6" />
        <circle cx="54" cy="536" r="6" />
        <circle cx="70" cy="536" r="6" />
        <rect width="53" height="14" y="556" rx="0" ry="0" />
        <rect width="147" height="16" y="586" rx="0" ry="0" />
      </g>
    </svg>
  </div>
);
