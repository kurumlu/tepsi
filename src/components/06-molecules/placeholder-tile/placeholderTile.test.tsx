import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';

import { PlaceholderTile } from './PlaceholderTile';

describe('<PlaceholderTile />', () => {
  const component: ShallowWrapper = shallow(<PlaceholderTile />);

  it('renders <PlaceholderTile /> component', () => {
    expect(component).toMatchSnapshot();
  });
});
