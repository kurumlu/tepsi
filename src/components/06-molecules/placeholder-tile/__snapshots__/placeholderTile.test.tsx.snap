// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlaceholderTile /> renders <PlaceholderTile /> component 1`] = `
<div
  className="sd-m-product-tile"
>
  <svg
    preserveAspectRatio="none"
    role="img"
    style={
      Object {
        "height": "auto",
        "width": "100%",
      }
    }
    viewBox="0 0 316 620"
  >
    <path
      d="M-1-1h1002v1002H-1z"
      fill="#fff"
    />
    <g
      fill="#f3f3f3"
    >
      <rect
        height="474"
        rx="5"
        ry="5"
        width="316"
      />
      <rect
        height="20"
        rx="0"
        ry="0"
        width="186"
        y="480"
      />
      <circle
        cx="6"
        cy="536"
        r="6"
      />
      <circle
        cx="22"
        cy="536"
        r="6"
      />
      <circle
        cx="38"
        cy="536"
        r="6"
      />
      <circle
        cx="54"
        cy="536"
        r="6"
      />
      <circle
        cx="70"
        cy="536"
        r="6"
      />
      <rect
        height="14"
        rx="0"
        ry="0"
        width="53"
        y="556"
      />
      <rect
        height="16"
        rx="0"
        ry="0"
        width="147"
        y="586"
      />
    </g>
  </svg>
</div>
`;
